Makefile.am

```
@@ -217,11 +216,29 @@ precomp: $(PRECOMP)
 # e.g., after `make maintainer-clean`).
 BUILT_SOURCES = $(PRECOMP)
 
-maintainer-clean-local: clean-precomp
-
+.PHONY: clean-precomp
 clean-precomp:
 	rm -f $(PRECOMP)
+maintainer-clean-local: clean-precomp
+
+### Pregenerated test vectors
+### (see the comments in the previous section for detailed rationale)
+TESTVECTORS = src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.h
+
+src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.h:
+	mkdir -p $(@D)
```

For more robustness, could consider
```suggestion
	$(MKDIR_P) $(@D)
```

with the [`AC_PROG_MKDIR_P`](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/autoconf.html#index-AC_005fPROG_005fMKDIR_005fP-1) macro in the `configure.ac`?

---

As I understand the history of this, the reason why this macro exists is that some implementations of `mkdir -p` on some old systems are not thread safe. And that's a problem in a parallel make, where multiple make processes try to create the same directory (see https://lists.gnu.org/r/automake/2006-04/msg00084.html).

I don't see how this could be a problem here, given that there's only a single recipe in the Makefile that creates this directory. That's why I think it's better to stick to `mkdir -p` here for simplicity.